Gemfile.lock

```diff
@@ -148,7 +166,8 @@ DEPENDENCIES
   pg (~> 0.18)
   puma (~> 3.7)
   rails (~> 5.1.0)
-  rubocop
+  rspec-rails
```

Comment:
👀

Reply:
Done!
I forgot to delete the gem, but now that is ok